src/share/poudriere/jail.sh

```diff
@@ -198,6 +198,16 @@ update_jail() {
 	msg "Upgrading using ${METHOD}"
 	case ${METHOD} in
 	ftp|http|ftp-archive)
+		# In case we use FreeBSD dists and TORELEASE is present, check if it's a release branch.
+		if [ ! -z ${TORELEASE} ]; then
+		  case ${TORELEASE} in
+		    *-ALPHA*|*-CURRENT|*-PRERELEASE|*-STABLE)
+			msg_error "Only release branches are supported by freebsd-update(8)."
+			msg_error "Please try to upgrade to a new BETA, RC or RELEASE version."
+			exit 1 ;;
```

Comment:
Please put `;;` on next line to follow other style for multi-line case statements.